spaces/dawn/index.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Dawn",
+  "network": "1",
+  "symbol": "DAWN",
+  "skin": "dawn",
```

You will need to add domain field to make your subdomain work.
```suggestion
  "skin": "dawn",
  "domain": "governance.dawn.org",
```

---

Thank you!

---

Added domain - not sure in `domains.json` is also needed.